
Mon, 27 Dec 2010 20:01:41 GMT

COMMAND LINE:
/usr/local/src/pypy-1.4.1-linux64/bin/pypy regexdna.pypy 0 < regexdna-input50000.txt

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "app_main.py", line 53, in run_toplevel
  File "regexdna.pypy", line 9, in <module>
    from multiprocessing import Pool
ImportError: No module named multiprocessing
